### ISSUE TITLE
Cannot assign shortcut for "Toggle Fullscreen" to any other function

### ISSUE THREAD

**Issue by jmoerschbach** [widelands/widelands#5001]
Trying to assign Ctrl+F (toggle fullscreen default shortcut) to any other shortcut action available toggles fullscreen instead of showing the dialog which informs about conflicting shortcuts

**To Reproduce**
1. Go to Options
2. Click on Edit keyboard shortcuts
3. Click on Take screenshot (F11)
4. Assign Ctrl+F
5. Fullscreen is toggled

**Expected behavior**
Infobox informing about conflicting screenshot should be shown like for all other shortcut conflicts

**Screenshots**

**Version:**
 - OS: [e.g. Ubuntu 20-04]
 - Widelands Version: 1.1~git25396[42b5f0e@master]

**Comment by tothxa** [comment]
Please check whether the solution in https://github.com/tothxa/widelands/commit/038aaf7afe2ccd52fec8cf0dffcfd01f49c273cb is acceptable.
